torchci/lib/drciUtils.ts

```diff
@@ -323,6 +329,13 @@ export async function hasSimilarFailures(
     if (!isEligibleCommit) {
       continue;
     }
+    if (
```

Comment:
I realize that let's not do the check here as it wastes a query to ElasticSearch.  You can add this check right at the begin of the function around line 258.  The failure string for the job is available there as `job.failure_captures`.